deploy/lib/server_config.rb

```diff
@@ -949,7 +955,7 @@ def load_data(dir, options = {})
     batch = @environment != "local" && batch_override.blank? || batch_override.to_b
 
     options[:batch_commit] = batch
-    options[:permissions] = permissions(@properties['ml.app-role'], Roxy::ContentCapability::ER) unless options[:permissions]
+    options[:permissions] = permissions(@properties['ml.app-role'], Roxy::ContentCapability::ERU) unless options[:permissions]
```

Comment:
Looks like there are a few redundant changes in this PR. Did you start off with upstream/dev, and did you fetch and rebase against that as well?